beacon-chain/core/blocks/payload.go

```diff
@@ -99,12 +91,17 @@ func ExecutionEnabled(st state.BeaconState, body block.BeaconBlockBody) (bool, e
 }
 
 // IsExecutionEnabledUsingHeader returns true if the execution is enabled using post processed payload header and block body.
-// This is an optimized version of ExecutionEnabled where beacon state is not required as an argument.
+// This is an optimized version of IsExecutionEnabled where beacon state is not required as an argument.
 func IsExecutionEnabledUsingHeader(header *ethpb.ExecutionPayloadHeader, body block.BeaconBlockBody) (bool, error) {
 	if !isEmptyHeader(header) {
 		return true, nil
 	}
-	return ExecutionBlock(body)
+	return IsExecutionBlock(body)
+}
+
+// IsPreBellatrixVersion returns true if input version is before bellatrix fork.
+func IsPreBellatrixVersion(v int) bool {
+	return v == version.Phase0 || v == version.Altair
```

Comment:
```suggestion
	return v < version.Bellatrix
```
Does this work in Golang? it would in `C`.